plugins/tech-radar/src/components/RadarLegend/RadarLegend.tsx

```diff
@@ -77,122 +84,138 @@ const useStyles = makeStyles<Theme>(() => ({
   entryLink: {
     pointerEvents: 'none',
   },
+};
+
+const useStyles = makeStyles<Theme>(() => ({
+  ringStyles,
+  quadrantStyles,
+  entryStyle,
```

Comment:
Not sure I'm a fan of this pattern given that you frequently want to access the theme quite often in styles. Have you used it before in a codebase with lots of theme access and is there some way to handle that nicely?

Reply:
Oh yeah ok ! I left the code as it was ! I just found styles a bit big and wanted to split them into their own scope. I will try to find if there is a way to handle that 👍